package.json

```diff
@@ -15,7 +16,6 @@
     "babel-brunch": "~6.0.0",
     "uglify-js-brunch": "~2.0.1"
   },
-  "files": ["README.md", "LICENSE.md", "package.json", "priv/static/phoenix.js", "web/static/js/phoenix.js"],
```

Comment:
> That concern is imagined, IMO, but fine.

Btw, I just tried it locally, and removing the "files" configuration includes all of the files in lib, which are Elixir files, in the npm package.